ndmg/utils/reg_utils.py

```diff
@@ -238,9 +240,19 @@ def t1w_skullstrip(t1w, out):
         path for the input t1w image file
     out : str
         path for the output skull-stripped image file
+    skull : int, optional
+        skullstrip parameter pre-set. Default is 0.
     """
-    
-    cmd = "3dSkullStrip -prefix {} -input {}".format(out, t1w)
+    if skull == 1:
+        cmd = "3dSkullStrip -prefix {} -input {} -shrink_fac_bot_lim 0.6 -ld 50".format(out, t1w)
+    elif skull == 2:
+        cmd = "3dSkullStrip -prefix {} -input {} -shrink_fac_bot_lim 0.3 -ld 50".format(out, t1w)
+    elif skull == 3:
+        cmd = "3dSkullStrip -prefix {} -input {} -no_avoid eyes -ld 50".format(out, t1w)
+    elif skull == 4:
+        cmd = "3dSkullStrip -prefix {} -input {} -push_to_edge -ld 50".format(out, t1w)
+    else:
+        cmd = "3dSkullStrip -prefix {} -input {} -ld 50".format(out, t1w)
```

Comment:
Generally I think it's better practice to use labels instead of integer indices (e.g., `excess_tissue` rather than `1`) - more clear that way.

Up to your discretion whether you want to make this change though

Reply:
Actually I'm on the fence about this one-- I'd say technically yes, but if the is a natural ordering to what you're doing here (i.e. 2>1 and 3>2 has meaning), then integers may be more appropriate.